src/traces/scatter/calc.js

```diff
@@ -51,23 +51,13 @@ function calc(gd, trace) {
         calcAxisExpansion(gd, trace, xa, ya, x, y, ppad);
     }
 
-    var hasPeriodX = !!trace.xperiodalignment;
```

Comment:
interesting that this PR is mostly removals :)

Reply:
Less is more :)